config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -281,6 +281,44 @@ presubmits:
         env:
         - name: GOPATH
           value: /go
+  - name: pull-kubernetes-node-kubelet-serial-containerd
```

Comment:
Do we also want to add a periodic? I don't think one exists right now

Reply:
not yet, let's get the presubmit working first.